viewing_party/party.py

```diff
@@ -1,23 +1,193 @@
 # ------------- WAVE 1 --------------------
+import copy
+from collections import Counter
 
+#1
 def create_movie(title, genre, rating):
-    pass
+    movie_dic = {}
+    if not title or not genre or not rating:
+        return None      
+    else: 
+         movie_dic["title"] = title
+         movie_dic["genre"] = genre
+         movie_dic["rating"] = rating
+         return movie_dic
+#2
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+#3
+def add_to_watchlist(user_data,movie):
+   user_data["watchlist"].append(movie)
+   return user_data
+
+#4 
+def watch_movie(user_data, title):
+    user_data_copy = copy.deepcopy(user_data)

```

Comment:
We actually do want to change `user_data` directly so we don't need a copy of it.
```suggestion

```